src/core/StackExchange.Redis.Extensions.Core/StackExchange.Redis.Extensions.Core.csproj

```diff
@@ -8,8 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
-    <PackageReference Include="StackExchange.Redis" Version="2.7.17" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Condition="'$(TargetFramework)' == 'net9.0'" Version="9.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Condition="'$(TargetFramework)' == 'net8.0'" Version="8.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Condition="'$(TargetFramework)' == 'netstandard2.1'" Version="8.0.*" />
+    <PackageReference Include="StackExchange.Redis" Version="2.8.16" />

```

Comment:
Whether `Microsoft.Extensions.Logging.Abstractions` should be removed as `StackExchange.Redis` has already claim it as a dependency?

@imperugo @thompson-tomo

Reply:
I already removed it locally, I'll push later today some code that runs on .NET 9 RC2 (I've also refactored the analyzer part)